pkg/controller/config/config_controller.go

```diff
@@ -225,21 +254,86 @@ func (r *ReconcileConfig) reconcileInstall(req reconcile.Request, res *op.Config
 		_ = r.updateStatus(res, op.ConfigCondition{
 			Code:    op.ErrorStatus,
 			Details: err.Error(),
-			Version: tektonVersion})
+			Version: TektonVersion})
+		return reconcile.Result{}, err
+	}
+
+	err := r.updateStatus(res, op.ConfigCondition{Code: op.InstallingStatus, Version: TektonVersion})
+	if err != nil {
+		log.Error(err, "failed to set installing pipeline status")
+		return reconcile.Result{Requeue: true}, err
+	}
+
+	log.Info("successfully applied pipeline resources")
+	return reconcile.Result{}, nil
+}
+
+func (r *ReconcileConfig) verifyDeployments(logger logr.Logger, res *op.Config) (bool, error) {
+	log.Info("verifying pipeline deployments")
+
+	deployments := appsv1.DeploymentList{}
+	listOpts := client.ListOptions{
```

Comment:
wouldn't all deployments have the owner-ref set which we should/can make use of?

Reply:
technically we can do it there no out of box API or declarative way of doing it